engine/src/test/java/io/zeebe/engine/job/JobTimeOutTest.java

```diff
@@ -75,18 +66,18 @@ public void shouldNotTimeOutIfDeadlineNotExceeded() {
   @Test
   public void shouldNotTimeOutIfJobCompleted() {
```

Comment:
These kind of tests are terrible. They will take at least the max timeout time of the RecordingExporter, which is currently 5 sec.

Reply:
I know but it was already there and I didn't want to remove it unilaterally. Should I?

Reply:
Sorry, didn't wanted to blame you was more a comment :sweat_smile: Yes maybe better. I assume in the JobStateTest it is tested that completed or failed jobs are not available anymore.

Reply:
No worries. Sure thing, I'll remove it

Reply:
This class has several tests that use `assertNoMoreJobsReceived`, which slows them considerably. Should I removed them as well? I'd be removing half of the remaining tests

Reply:
Remove them, please. Except you see any value in them. I will take a look on the second review as well